encoding/thrift/thriftrw-plugin-yarpc/fx.go

```diff
@@ -33,10 +33,20 @@ const fxTemplate = `
 <$pkgname := printf "%sfx" (lower .Name)>
 package <$pkgname>
 
-<$yarpc  := import "go.uber.org/yarpc">
+<$transport := import "go.uber.org/yarpc/api/transport">
 <$thrift := import "go.uber.org/yarpc/encoding/thrift">
 <$client := import .ClientPackagePath>
 
+// Params defines the dependencies for yarpc.
```

Comment:
Not for YARPC but for the `<.Name> client`. See docs for `Client` for an example.